packages/headless/src/features/case-assist/case-assist-analytics-actions.ts

```diff
@@ -68,12 +68,19 @@ export const logClassificationClick = (classificationId: string) =>
       })
   )();
 
-export const logDocumentSuggestionClick = (suggestionId: string) =>
+export const logDocumentSuggestionClick = (
+  suggestionId: string,
+  fromQuickview = false
```

Comment:
Curious why do we need to distinguish between fromQuickview or otherwise? I'm guessing from an analytics/ml perspective, this detail is important?

If it is, could it be clearer if we had a separate action? e.g. `logQuickviewDocumentSuggestionClick`

I'm also not clear on when this should be used:
- Is it when some opens the quickview?
- Is it when some clicks the document title inside an open quickview?

Separate action would allow for dedicated documentation to explain,

Reply:
Basically we want to distinguish between when the the user clicks on the title of a document suggestion and when the user clicks on the quickview button of a document suggestion, you guessed it right from an analytics/ml perspective we want to differentiate  the two events, a differentiation which was considered important in a case assist use case but you absolutely right having a separate action for the event related to the quickview will be much better 👌